### PR TITLE
update sdk to v1alpha2 and handle changes for new knative and types

### LINKED ISSUE #249: Upgrade openapi and python sdk once knative upgraded to v1beta1
/kind feature

**Describe the solution you'd like**
We need to upgrade openapi and python sdk once knative upgraded to v1beta1 (after the PR merged #238 ), since there are some changes in the `kfservice_types.go `

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:

Continue working for PR #272...  Apologize, I made a mistake (frustrated rebasing) and caused that PR closed abnormally and hard to recover, so send new one to finish that.  

Due to the knative is updated in the PR #251 , there are some changes in the knative and also has updates in kfservce_type.go, so we need to update the openapi, swagger file and python client.

**Which issue(s) this PR fixes** :
Fixes #249  #294 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/295)

### CODE REVIEW

File: python/kfserving/kfserving/models/knative_dev_pkg_apis_condition.py [hunk 1]
```diff
@@ -16,10 +16,10 @@
 
 import six
 
-from kfserving.models.knative_volatile_time import KnativeVolatileTime  # noqa: F401,E501
+from kfserving.models.knative_dev_pkg_apis_volatile_time import KnativeDevPkgApisVolatileTime  # noqa: F401,E501
 
 
-class KnativeCondition(object):
+class KnativeDevPkgApisCondition(object):
```
Comment (yuzisun) [inline python/kfserving/kfserving/models/knative_dev_pkg_apis_condition.py]: should we still keep the name `KnativeCondition`?

File: python/kfserving/kfserving/models/knative_dev_pkg_apis_volatile_time.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from kfserving.models.v1_time import V1Time  # noqa: F401,E501
 
 
-class KnativeVolatileTime(object):
+class KnativeDevPkgApisVolatileTime(object):
```
Comment (yuzisun) [inline python/kfserving/kfserving/models/knative_dev_pkg_apis_volatile_time.py]: same here

File: python/kfserving/kfserving/models/knative_dev_pkg_apis_condition.py (hunk 1 above)
Comment (jinchihe) [reply python/kfserving/kfserving/models/knative_dev_pkg_apis_condition.py]: Agree with you, updated, thanks!

File: python/kfserving/kfserving/models/knative_dev_pkg_apis_volatile_time.py (hunk 2 above)
Comment (jinchihe) [reply python/kfserving/kfserving/models/knative_dev_pkg_apis_volatile_time.py]: Agree with you, updated, thanks!

**Comment by jinchihe** [comment]
Since the KFServing has been upgraded from from v1alpha1 to v1alpha2 , also upgraded SDK accordingly in the PR. Thanks.

/cc @yuzisun @ellis-bigelow @animeshsingh

File: cmd/spec-gen/main.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 
 	"github.com/go-openapi/spec"
-	kfsvcv1alpha1 "github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2"
```
Comment (ellis-bigelow) [inline cmd/spec-gen/main.go]: Should we just import these as kfserving? Simplifies imports.

Comment (jinchihe) [reply cmd/spec-gen/main.go]: Updated, thanks.

**Comment by jinchihe** [comment]
> What is the patch.tar.gz file?

Removed, that should be not check in. Thanks for the catching @ellis-bigelow

**Comment by yuzisun** [comment]
thanks @jinchihe! I apologize that you may need to regenerate this again after the real v1alpha2 changes..

/lgtm

**Comment by jinchihe** [comment]
> thanks @jinchihe! I apologize that you may need to regenerate this again after the real v1alpha2 changes..
> 
Glad to do that. :-) Thanks.

**Pull request merged**